package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.14.0",
   "description": "Sushi Unshortens Short Hand Inputs (FSH Compiler)",
   "scripts": {
-    "build": "rm -rf dist && tsc && cp -r src/ig/files dist/ig/files",
+    "build": "rm -rf dist && tsc && cp -r src/ig/files dist/ig/files && cp src/import/generated/*.d.ts dist/import/generated/",
     "build:watch": "tsc -w",
     "build:grammar": "bash antlr/gradlew -p antlr generateGrammarSource",
```

Comment:
In my testing, a rebuild of the grammar wipes out the `generated/*.d.ts` files. Probably easiest just to move those elsewhere, and would make sense as well, since they aren't generated files anyway.

Reply:
That does seem unfortunate... I wonder if we can keep these type files somewhere under the `antlr` folder and then update `build.gradle` to copy them over to `src/import/generated` after running the ANTLR generation?

Also... was fixing these file formulaic?  Like... would it be possible to write an algorithm that lets TSC generate the bad files but then automagically fixes them?  That would help us avoid the issue of the type files getting out of sync w/ the grammar...

Reply:
I like this idea best, so I'll add something in the build directory to do this.